tools/btfgen.sh

```diff
@@ -0,0 +1,46 @@
+
+#!/bin/bash
+
+set -e
+set -x
+
+LIBBPFTOOLS="${LIBBPFTOOLS:-../bcc/libbpf-tools/.output/}"
+BTFHUB="${BTFHUB:-../btfhub/}"
+ARCH=x86_64
+BTFGEN=${BTFHUB}/tools/bin/btfgen.x86_64
+INPUT=${BTFHUB}/archive/
+OUTPUT=/tmp/btfs
+
+if [ ! -d ${INPUT} ]; then
+    echo "error: could not find input directory."
+    exit 1
+fi
+
+if [ ! -x "${BTFGEN}" ]; then
+    echo "error: could not find btfgen tool"
+    exit 1
+fi
+
+if [ ! -d "${LIBBPFTOOLS}" ]; then
+    echo "error: libbpftools folder not found"
+    exit 1
+fi
+
+mkdir -p ${OUTPUT}
+
+for dir in $(find ${INPUT} -iregex ".*${ARCH}.*" -type d | sort -u); do
+    # skip folder without btf files
+    ls ${dir}/*.btf > /dev/null 2>&1 || continue
+
+    dirnoprefix=${dir#"$INPUT"}/
+    mkdir -p ${OUTPUT}/${dirnoprefix}
+
+    ${BTFGEN} -i ${dir} -o ${OUTPUT}/${dirnoprefix}                 \
+        --object ${LIBBPFTOOLS}/bindsnoop.bpf.o                     \
+        --object ${LIBBPFTOOLS}/execsnoop.bpf.o                     \
+        --object ${LIBBPFTOOLS}/mountsnoop.bpf.o                    \
+        --object ${LIBBPFTOOLS}/opensnoop.bpf.o                     \
+        --object ${LIBBPFTOOLS}/tcpconnect.bpf.o                    \
+        --object ./pkg/gadgets/seccomp/tracer/bpf/seccomp.o         \
```

Comment:
Yes, that's right. Those don't work with BTFGen because they required native BTF support in the kernel, so it doesn't make sense to include them here.